controllers/tempest_controller.go

```diff
@@ -343,11 +343,11 @@ func (r *TempestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (re
 	//                 It will simplify the logic and duplicite code (Tempest vs Tobiko)
 	if externalWorkflowCounter < len(instance.Spec.Workflow) {
```

Comment:
You want to add a similar piece of code into the AnsibleTest controller as well. Right now `NodeSelector` and `Tolerations` do not work in `workflow` section of `AnsibleTest CR`.

Reply:
this is not actually needed for the ansibleTest as far as I see it but I can add it for uniformity.